resources/variants/felixtec4_0.25.inst.cfg

```diff
@@ -6,8 +6,9 @@ definition = felixtec4dual
 [metadata]
 author = kerog777
 type = variant
-setting_version = 1
+setting_version = 4
 
 [values]
+hardware_type = nozzle
```

Comment:
The `hardware_type = nozzle` must be in the `[metadata]` section, not in the `[values]`. You can take a look at some of the built-in variants, such as https://github.com/Ultimaker/Cura/blob/master/resources/variants/ultimaker_s5_aa04.inst.cfg